lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -8,7 +8,7 @@ class DisabledConfigFormatter < BaseFormatter
       HEADING =
         ['# This configuration was generated by',
          '# `%s`',
-         "# on #{Time.now} using RuboCop version #{Version.version}.",
+         "# %susing RuboCop version #{Version.version}.",
```

Comment:
New cop idea - string array followed by `.join("\n")` should be replaced with a heredoc. 😄 

/cc @Drenmi @garettarrowood @pocke @koic

Reply:
Fixed in most recent push